Robinhood/Robinhood.py

```diff
@@ -73,11 +56,75 @@ class Robinhood:
     logger = logging.getLogger('Robinhood')
     logger.addHandler(logging.NullHandler())
 
+    from . import exceptions
```

Comment:
import inside class is not PEP

Reply:
I know, but frankly this whole lib should not be a Class.  It should follow a session model, such as requests does where it is easier to have utility functions and other things in separate files.